/usr/bin/ant will put dozens of jars on the classpath, including a
rhino.jar that breaks the build

--- rhino/build.xml.orig	2021-11-11 17:25:07.284267174 +0100
+++ rhino/build.xml	2021-11-11 17:25:38.689242510 +0100
@@ -33,6 +33,7 @@
     <property file="apiClasses.properties"/>
     <property name="docsrc.dir" value="docs"/>
     <property name="dist.docsrc.dir" value="src/docs"/>
+    <property name="build.sysclasspath" value="ignore"/>
   </target>
 
   <target name="init" depends="properties">
